app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ReservationsController < ApplicationController
+  def new
+  end
+
+  def create
+  end
+
+  #def index
+    #@reservations = User.find(current_user.id).reservations
+  #end
+
+  #def show
+    #@reservation = Reservation.find(params[:id])
```

Comment:
Incorrect indentation detected (column 4 instead of 2).<br>Missing space after #.